invenio_oaiserver/ext.py

```diff
@@ -120,19 +120,18 @@ def init_config(self, app):
             app.config.get('THEME_SITENAME',
                            'Invenio-OAIServer'))
 
+        for k in dir(config):
+            if k.startswith('OAISERVER_'):
+                app.config.setdefault(k, getattr(config, k))
+
         # warn user if ID_PREFIX is not set
-        if 'OAISERVER_ID_PREFIX' not in app.config:
+        if app.config.get('OAISERVER_ID_PREFIX') is None:
             import socket
             import warnings
 
-            app.config.setdefault(
-                'OAISERVER_ID_PREFIX',
-                'oai:{0}:recid/'.format(socket.gethostname()))
+            app.config['OAISERVER_ID_PREFIX'] = \
+                '1'.format(socket.gethostname())
```

Comment:
Is this intentional?

Reply:
Nope, it's a mistake (which doesn't cause tests to fail though...)